nuxt.config.js

```diff
@@ -1,38 +1,80 @@
+const env = require('dotenv').config().parsed
+
+if (typeof env.API_URL === 'undefined') {
+  throw new Error('API_URL enviroment variable must be defined.')
+}
+
 module.exports = {
-  env: require('dotenv').config().parsed,
+  env,
   /*
-  ** Headers of the page
-  */
+   ** Headers of the page
+   */
   head: {
-    title: 'homepage-nuxtjs',
-    meta: [
-      { charset: 'utf-8' },
-      { name: 'viewport', content: 'width=device-width, initial-scale=1' },
-      { hid: 'description', name: 'description', content: 'Nuxt.js project' }
+    title: 'Koło Naukowe IT - Politechnika Krakowska',
+    meta: [{
+      charset: 'utf-8'
+    },
+    {
+      name: 'viewport',
+      content: 'width=device-width, initial-scale=1'
+    },
+    {
+      hid: 'description',
+      name: 'description',
+      content: 'Nuxt.js project'
+    }
     ],
     script: [],
-    link: [
-      { rel: 'icon', type: 'image/x-icon', href: '/favicon.ico' },
-      { rel: 'stylesheet', href: 'https://cdn.rawgit.com/konpa/devicon/df6431e323547add1b4cf45992913f15286456d3/devicon.min.css' },
-      { rel: 'stylesheet', href: 'https://fonts.googleapis.com/css?family=Open+Sans:300,400,600&amp;subset=latin-ext' }
+    link: [{
+      rel: 'icon',
+      type: 'image/x-icon',
+      href: '/favicon.ico'
+    },
+    {
+      rel: 'dns-prefetch',
+      href: '//cdn.rawgit.com'
+    },
+    {
+      rel: 'dns-prefetch',
+      href: '//fonts.googleapis.com'
+    },
+    {
+      rel: 'preconnect',
+      href: env.API_URL,
+      crossorigin: 'anonymous'
+    },
+    {
+      rel: 'prefetch stylesheet',
+      href: 'https://cdn.rawgit.com/konpa/devicon/df6431e323547add1b4cf45992913f15286456d3/devicon.min.css'
+    },
+    {
+      rel: 'prefetch stylesheet',
```

Comment:
The better option is to use `{ rel: 'preload', as 'font', href: 'https...' }` insstead of `prefetch`.
I have made two tests:
1. prefetch 
![image](https://user-images.githubusercontent.com/23239447/33895591-202673d0-df61-11e7-9f57-93f8629e904e.png)
2. preload 
![image](https://user-images.githubusercontent.com/23239447/33895630-34b75b7a-df61-11e7-8812-6808d05f4315.png)
> Preload is an early fetch instruction to the browser to request a resource needed for a page (key scripts, Web Fonts, hero images).

> Prefetch serves a slightly different use case — a future navigation by the user (e.g between views or pages) where fetched resources and requests need to persist across navigations. If Page A initiates a prefetch request for critical resources needed for Page B, the critical resource and navigation requests can be completed in parallel. If we used preload for this use case, it would be immediately cancelled on Page A’s unload.

Reply:
Yeah that's much better